### PR TITLE
Covering up a possible error : db.conf NOT FOUND

### PR DESCRIPTION
**Pull request by humbleOldSage**
Setting the repository up for the first time might cause this error to pop up. A solution was proposed by Shankari earlier in Teams chat . Just migrating it here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,8 +18,8 @@ it needs to be included while running them.
     ```
     $ source setup.sh
     ```
-
-1. If you have, activate
+ 
+ 1. If you have, activate
```
Comment (shankari) [inline README.md:22]: Please don't include extraneous whitespace changes. It messes up the blame and makes everything harder to work with in the future.

File: README.md [hunk 2]
```diff
@@ -38,6 +38,37 @@ $ cp -r ../conf .
...
+>    $ env
+>    ```
+>
+>
+>* Move the conf files AGAIN using :
+>
+>    ```console
+>    cp -r $EMISSION_SERVER_HOME/conf .
+>    ```
+>
```
Comment (shankari) [inline README.md:68]: This is not moving the files, it is copying them.

**Review by shankari — CHANGES_REQUESTED** [review]
At a high level, I don't understand why you have to replicate instructions. There are already instructions to set the EMISSION_SERVER_HOME on line 12. Why not clarify those instructions instead of creating a whole new set of instructions? This is likely to be even more confusing - e.g. do I have to set `EMISSION_SERVER_HOME` twice? 

Similarly, the existing instructions already say (on lines 31-32), that you should `cd` into the `<eval_folder>` before copying the conf files over. Why not clarify that instead of saying that you need to copy files over again?

File: README.md (hunk 1 above)
Comment (humbleOldSage) [reply README.md:22]: I'll correct it right away.

**Comment by humbleOldSage** [comment]
Rather than repeating the instructions, I have now included a check to ensure that right path was added in the environment variables.

**Comment by shankari** [comment]
@humbleOldSage this looks good, but incomplete. What about highlighting that you have the copy the `conf` directory **after** `cd` into the correct directory?

**Comment by humbleOldSage** [comment]
Added an example of what an '<eval_folder>' could be since this was the part where I felt a bit confused and created my own empty eval folder.

File: README.md [hunk 3]
```diff
@@ -13,19 +13,25 @@ it needs to be included while running them.
...
+    and ensure ```ENV_SERVER_HOME``` is present in the list and has the right path (as mentioned above).
+    
 1. If you haven't setup before, set up the evaluation system
 
     ```
     $ source setup.sh
     ```
-
-1. If you have, activate
+ 1. If you have, activate
```
Comment (shankari) [inline README.md:27]: Extraneous whitespace, please fix.

**Review by shankari — CHANGES_REQUESTED** [review]
Almost there! I will squash merge this this experimental change has a kind of messy set of commits. This means that your master will have what appear to be unresolved commits. Please handle this properly (e.g. by deleting master/deleting the fork) and make sure to submit from non-master branches in the future.

**Comment by humbleOldSage** [comment]
I'll be careful with the branches moving forward .

**Review by shankari — APPROVED** [review]

**Comment by shankari** [comment]
Congratulations on your first merged PR! 🎉

**Pull request merged**